lib/features/support/app_automate_driver.rb

```diff
@@ -89,6 +100,39 @@ def send_keys_to_element(element_id, text)
     find_element(@element_locator, element_id).send_keys(text)
   end
 
+  # Determines and returns sensible project, build, and name capabilities
+  #
+  # @param target_device [String] the device in the device list being targeted
+  #
+  # @return [Hash] A hash containing the 'project', 'build', and 'name' capabilities
+  def project_name_capabilities(target_device)
```

Comment:
@twometresteve can you have a quick look at this method to make sure you're happy with how it's generating the build names?

Reply:
Looks fine as a piece of code.  I just don't have good enough understand of how it will manifest itself within the build and if space separated strings will be an issue (or example), but happy for it to be merged and corrected later if necessary.